OneCommunityGlobal/HighestGoodNetworkApp
# Xiao Improve new-user Team and Project setup efficiency

# Description
1. (PRIORITY MEDIUM) Jae: Improve new-user Team and Project setup efficiency
    1. Admin/Owner Login → Other Links → User Management → Create New User
    2. Right now, when I create a new user I have more clicks than I need for “Assign Project” and “Assign Team”
    3. What I’d like instead of what is shown below is to change the “Project” and “Team” tabs into “Assign Project’ and “Assign Team” buttons with the functions those buttons currently perform… eliminating the need for clicking the tabs and moving the information in those tabs so it is immediately visible with scrolling. 
![unknown](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/6b51e5dc-911a-46a5-9477-3cf7b8520b35)

## Main changes explained:
Removed nav component in UserProfileAdd.jsx and refactored the code to make "Assign Team" appear directly visible under "Assign Project".

## How to test:
    1. check into current branch
    2. do npm run start:local to run this PR locally
    3. log in as Admin/Owner, go to Other Links → User Management → Create New User
    5. verify "Assign Team" is immediately visible with scrolling

## Screenshots or videos of changes:
After:
<img width="695" alt="Pasted Graphic" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/a65357af-3202-4d24-9f9f-321b22ee2c1a">

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/063c341b-8569-4ca7-8df8-31081bf01679

## Note:
Test to make sure the functionalities for "Assign Project" and "Assign Team" remain unchanged.
